src/openrct2/network/NetworkBase.h

```diff
@@ -113,19 +123,42 @@ class NetworkBase : public OpenRCT2::System
 
 public: // Client
     void Reconnect();
-    int32_t GetMode();
+
+    constexpr int32_t GetMode() const noexcept
+    {
+        return mode;
+    }
```

Comment:
I would prefer we do less actual code in headers, what is the reason you moved it? Also you won't be getting any benefits out of marking this constexpr as it will never be able to fully evaluate the entire context. Constexpr makes only sense if you can evaluate the entire state at compile time.

Reply:
You're right.
I misunderstood about what `constexpr` is for.
Reverted all changed regarding to constexpr and code in header!